entrypoint.sh

```diff
@@ -1,13 +1,6 @@
 #! /usr/bin/env bash
-
 set -e
 
-echo '### `crystal tool format --check`'
-crystal tool format --check
-
-echo '### `ameba`'
-crystal lib/ameba/bin/ameba.cr
-
```

Comment:
I understand you are replacing this with the `ci.yml` commands but this means when we run `./test` locally, `crystal tool format` and `ameba` will not be run. Is this ok?

Reply:
good point. i think i'll put it behind an env var.